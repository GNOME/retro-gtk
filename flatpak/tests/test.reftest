[Retro Reftest]
Path=/test
Core=/app/lib/libretro/test_libretro.so

[Options]
test_aspect=4:3;16:9;
test_samplerate=30000;20000;
test_analog_mouse=true;false;
test_analog_mouse_relative=false;true;
test_audio_enable=true;false;

[Frame 0]
State=Refresh
Video=test.png
