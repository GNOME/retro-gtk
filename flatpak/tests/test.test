[Retro Reftest]
Path=/test
Core=/app/lib/libretro/test_libretro.so

[Frame 0]
Tests=Run;Video;
Video=test.png
