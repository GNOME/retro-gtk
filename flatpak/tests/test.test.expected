/test/Boot: OK
/test/0/Run: OK
/test/0/Video: OK
/test/0/StateRefresh: OK
